File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i = 1; i <= n; i++) {
        	if(i % 2 != 0 && n % i == 0) {
        		answer++;
        	}
        }
        
        
        
        return answer;
    }
}

File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.py
def solution(n):
    answer = 0

    for i in range(1, n + 1):
        current_sum = 0

        for j in range(i, n + 1):
            current_sum += j
            
            if current_sum == n:
                answer += 1
                break
            elif current_sum > n:
                break

    return answer